### PR TITLE
Handle -1 case in solidity file mapping

### PR DESCRIPTION
**Pull request by norhh**

### CODE REVIEW

File: mythril/analysis/report.py [hunk 1]
```diff
@@ -103,6 +103,18 @@ def as_dict(self):
 
         return issue
 
+    def _add_false_positive_tags(self):
```
Comment (JoranHonig) [inline mythril/analysis/report.py]: Add internal compiler file information

File: mythril/analysis/report.py [hunk 2]
```diff
@@ -103,6 +103,18 @@ def as_dict(self):
...
+        """
+        Adds the false positive to description and changes severity to low
+        """
+        self.severity = "Low"
+        self.description_tail += (
+            " This happened in the internal compiler generated code, "
```
Comment (JoranHonig) [inline mythril/analysis/report.py]: I don't want to say this is a false positive, rather just something like: "This issue is reported for internal compiler generated code."
wdyt?

File: mythril/analysis/report.py [hunk 3]
```diff
@@ -115,6 +126,8 @@ def add_code_info(self, contract):
             self.filename = codeinfo.filename
             self.code = codeinfo.code
             self.lineno = codeinfo.lineno
+            if self.lineno is None:
```
Comment (JoranHonig) [inline mythril/analysis/report.py]:
```suggestion
            if codeinfo.solidity_file_idx == -1:
```
Should be equivalent, but I think this is a bit more self-explanatory

Comment (JoranHonig) [reply mythril/analysis/report.py]: what do you think?

Comment (norhh) [reply mythril/analysis/report.py]: I think codeinfo and codeinfo.filename attribute doesn't have solidity_file_idx.

Comment (JoranHonig) [reply mythril/analysis/report.py]: ah I had mistaken the type, what about `codeinfo.solc_mapping.solidity_file_idx`

**Pull request merged**